[2024-09-15T16:57:06.220+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-15T16:57:06.230+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_python_operator_hook.upload_to_bigquery_task manual__2024-09-15T16:57:04.525392+00:00 [queued]>
[2024-09-15T16:57:06.236+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_python_operator_hook.upload_to_bigquery_task manual__2024-09-15T16:57:04.525392+00:00 [queued]>
[2024-09-15T16:57:06.237+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-09-15T16:57:06.246+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_bigquery_task> on 2024-09-15 16:57:04.525392+00:00
[2024-09-15T16:57:06.252+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_python_operator_hook', 'upload_to_bigquery_task', 'manual__2024-09-15T16:57:04.525392+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/example_python_operator_hook.py', '--cfg-path', '/tmp/tmp30vtagew']
[2024-09-15T16:57:06.254+0000] {standard_task_runner.py:91} INFO - Job 179: Subtask upload_to_bigquery_task
[2024-09-15T16:57:06.255+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=400) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-15T16:57:06.255+0000] {standard_task_runner.py:63} INFO - Started process 402 to run task
[2024-09-15T16:57:06.280+0000] {task_command.py:426} INFO - Running <TaskInstance: example_python_operator_hook.upload_to_bigquery_task manual__2024-09-15T16:57:04.525392+00:00 [running]> on host 81bcc4ceaa16
[2024-09-15T16:57:06.338+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_python_operator_hook' AIRFLOW_CTX_TASK_ID='upload_to_bigquery_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-15T16:57:04.525392+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-15T16:57:04.525392+00:00'
[2024-09-15T16:57:06.339+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-15T16:57:06.348+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-15T16:57:06.349+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: upload_to_bigquery() missing 1 required positional argument: 'df'
[2024-09-15T16:57:06.355+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=example_python_operator_hook, task_id=upload_to_bigquery_task, run_id=manual__2024-09-15T16:57:04.525392+00:00, execution_date=20240915T165704, start_date=20240915T165706, end_date=20240915T165706
[2024-09-15T16:57:06.362+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 179 for task upload_to_bigquery_task (upload_to_bigquery() missing 1 required positional argument: 'df'; 402)
[2024-09-15T16:57:06.388+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-09-15T16:57:06.399+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-15T16:57:06.401+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
